docs/Demo.md

```diff
@@ -134,49 +229,203 @@ curl $(oc get route -n pelorus deploytime-exporter -o=template='http://{{.spec.h
 
 ### See the change to your todolist application
 You can now also check on your todolist application and see the updated text change "Add a todo"
+
 ![todolist-fixed](img/todolist_fixed.png)
 
 
 ## Troubleshooting
 
 If exporters are not functioning or deployed, no data will show up in the dashboard. It will look like the following:
+
 ![No-Data](img/pelorus-dashboard-no-data.png)
 
 * Please check the logs of exporter pod.
 
 An "idle" state could resemble:
+
 ![Idle-Data](img/pelorus-dashboard-idle-data.png)
 
 
-## Automated Demo
+# Mean Time to Restore and Change Failure Rate
```

Comment:
Same here, mention that in the previous section two metrics were measured and here another two are presented.

Reply:
k.. I *think* I added what your looking for, please double check